docs/conf.py

```diff
@@ -221,5 +221,12 @@
 
 linkcheck_ignore = [
     # Specific lines in Github blobs cannot be found by linkcheck.
-    r'https?://github\.com/.*?/blob/[a-f0-9]+/.+?#'
+    r'https?://github\.com/.*?/blob/[a-f0-9]+/.+?#',
+    # Links with images on index page have to point to html files
+    # which linkcheck cannot find.
+    'user-guide/data-structures.html',
```

Comment:
I am guessing you looked into how one could fix this, but found nothing... but just thought I would ask anyway, just to be sure